app/src/main/java/com/pysiak/simple/clock/ui/clock/Timezonedb.java

```diff
@@ -0,0 +1,173 @@
+package com.pysiak.simple.clock.ui.clock;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class Timezonedb {
```

Comment:
Move this class to com.pysiak.simple.clock.model.timezone package. Rename it to something more informative, for example PlaceTimeZone or CityTimeZone... Also you could remove unused fields/methods from this class if you want, it will also work (but you could also leave it as is)